superset/config.py

```diff
@@ -729,6 +729,7 @@ class D3TimeFormat(TypedDict, total=False):
 
 THUMBNAIL_CACHE_CONFIG: CacheConfig = {
     "CACHE_TYPE": "NullCache",
+    "CACHE_DEFAULT_TIMEOUT": 3600,
```

Comment:
we should be using `timedelta` here. Also, 3600 seems rather short for a thumbnail. Maybe a week? Or even a month?
```suggestion
    "CACHE_DEFAULT_TIMEOUT": int(timedelta(days=7).total_seconds()),
```

Reply:
lets go with a week and we can tweak it later if it's too short